src/testRunner/unittests/tsserver/getExportReferences.ts

```diff
@@ -0,0 +1,133 @@
+namespace ts.projectSystem {
+    describe("unittests:: tsserver:: getExportReferences", () => {
+        const mainTs: File = {
+            path: "/main.ts",
+            content: 'import { value, valueA, valueB, valueC, valueD } from "./mod";',
+        };
+        const modTs: File = {
+            path: "/mod.ts",
+            content: `export const value = 0;
+export const [valueA, valueB] = [0, 1];
+export const { valueC, valueD } = { valueC: 0, valueD: 1 };`,

```

Comment:
can you add a test that renames the destructured property `export const {foo: bar} = {foo: 2}` to make sure this works as expected